presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -178,10 +178,12 @@ this data must be mapped into columns to allow queries against the data.
     use any table definition files, but instead use the Presto
     :doc:`/functions/json` to parse the ``_message`` column which contains
     the bytes mapped into an UTF-8 string. This is, however, pretty
-    cumbersome and makes it difficult to write SQL queries.
+    cumbersome and makes it difficult to write SQL queries. This only works
+    when reading data.
 
 A table definition file consists of a JSON definition for a table. The
-name of the file can be arbitrary but must end in ``.json``.
+name of the file can be arbitrary but must end in ``.json``. This file lives
```

Comment:
Place the file in the directory configure with the ``kafka.table-description-dir`` property.